client/pages/fetch-data.jsx

```diff
@@ -25,21 +41,81 @@ export default class FetchData extends React.Component {
     this.willFetch(id);
   }
 
-  handleTeamDetails(id) {
-
-  }
-
   componentDidMount() {
 
     axios.request('/api/week-games/:date').then(response => {
       const fixtures = response.data;
       this.setState({
         fixturesList: fixtures,
-        isLoading: false
+        isLoading: false,
+        awayOdds: randomOdds(1, 5, 2),
+        homeOdds: randomOdds(1, 4, 2)
```

Comment:
As we discussed in chat, these odds should be stored in the database so that they persist.